dynparquet/schema.go

```diff
@@ -1077,6 +1092,56 @@ func (s *Schema) GetWriter(w io.Writer, dynamicColumns map[string][]string) (*Po
 	return pooled.(*PooledWriter), nil
 }
 
+type PooledParquetSchema struct {
+	pool   *sync.Pool
+	Schema *parquet.Schema
+}
+
+// GetParquetSortingSchema returns a parquet schema of the sorting columns and
+// the given dynamic columns.
+// The difference with GetDynamicParquetSchema is that non-sorting columns are elided.
+func (s *Schema) GetParquetSortingSchema(dynamicColumns map[string][]string) (*PooledParquetSchema, error) {
```

Comment:
This isn't being used anywhere that I can tell. Do we need it?

Reply:
It's used in `dynparquet/row.go`. See above.

Reply:
Doh! Github had collapsed that file